newton/examples/robot/example_robot_franka_teleop.py

```
@@ -0,0 +1,658 @@
+        if isinstance(key, int):
+            return key in (pyglet.window.key.LCTRL, pyglet.window.key.RCTRL)
+        return key.lower() in {"ctrl", "lctrl", "rctrl", "left_ctrl", "right_ctrl"}
+
+    def is_ctrl_active(self, modifiers: int | None = None) -> bool:
+        if modifiers is not None and modifiers & pyglet.window.key.MOD_CTRL:
+            return True
+        try:
+            return self.renderer.is_key_down(pyglet.window.key.LCTRL) or self.renderer.is_key_down(
+                pyglet.window.key.RCTRL
+            )
+        except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)